source/frontend/src/components/common/form/ContactInput/ContactInputView.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { Formik } from 'formik';
+import { IContactSearchResult } from 'interfaces';
+import noop from 'lodash/noop';
+import { mockLookups } from 'mocks/mockLookups';
+import { lookupCodesSlice } from 'store/slices/lookupCodes';
+import { act, render, RenderOptions, userEvent } from 'utils/test-utils';
+
+import ContactInputView, { IContactInputViewProps } from './ContactInputView';
+
+const setSelectedRows = jest.fn();
+const setShowContactManager = jest.fn();
+const clear = jest.fn();
+
+describe('ContactInputContainer component', () => {
```

Comment:
nit: should be "ContactInputView" component as this is the test for the view (not the container)